### DISCUSSION TITLE
Verified custom domain on Github Pages keeps on asking for a verification

### DISCUSSION THREAD

**Discussion by roccoan** [community/community#35805] [Other Feature Feedback, Questions, & Ideas]
Hello,

I am the owner of an organization for which I verified the domain as explained in 
https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages 
but when I try to add it to my repository, I keep getting this error:

> The custom domain `---.com` is already taken. If you are the owner of this domain, check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages for information about how to verify and release this domain. 

I have searched this issue on other threads and it says to wait for 7 days, but then I received an email from you saying that "Another account on the platform has a verified ---.com which you use for one of your GitHub Pages sites. Please verify your domain within 7 days by following our instructions before it becomes unverified".

How can I solve this issue?
How is it possible that someone else is using the domain, if I already verified it?

Thank you very much for your time

**Comment by airtower-luna** [comment]
> I am the owner of an organization for which I verified the domain […] but when I try to add it to my repository, I keep getting this error

Do I read correctly that you verified the domain for your organization, but are now trying to add it to a personal repository? If yes, that's probably the problem.

**Comment by roccoan** [comment]
No I'm not trying to add it to a personal repository.
I'm trying to add it to the organization's repository

Il mar 11 ott 2022, 19:00 Airtower ***@***.***> ha scritto:
